ca/server_test.go

```diff
@@ -1334,13 +1264,13 @@ func TestRootRotationReconciliationRace(t *testing.T) {
 		if !bytes.Equal(cluster.RootCA.CAKey, rotationKey) {
 			return errors.New("expected root key is wrong")
 		}
-		for _, secConfig := range secConfigs {
-			s, err := secConfig.RootCA().Signer()
+		for i, server := range append(otherServers) {
+			s, err := server.RootCA().Signer()
 			if err != nil {
 				return err
 			}
 			if !bytes.Equal(s.Key, rotationKey) {
-				return errors.New("all the sec configs haven't been updated yet")
+				return errors.Errorf("all the servers' root CAs haven't been updated yet: server %d", i)
```

Comment:
Have updated the message, although still just with the index, since the name might not be that useful in debugging which one failed.